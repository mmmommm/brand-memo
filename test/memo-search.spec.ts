import MemoSearch from '@/views/memo-search.vue'
import Vuetify from 'vuetify'

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe('memo-search.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  const wrapper = shallowMount(MemoSearch, {
    localVue,
    vuetify,
    data() {
      return {
        filteredMemo: [
          {
            code: 1111,
            slug: "11111111",
            name: "dummy",
            capitalization: 1,
            floating: 1,
            theme: "dummy",
            price: 1,
            url: "https://dummy.ac.jp",
            reason: "dummy"
          }
        ]
    }
    },
  })
  it('<MemoSearch />', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})