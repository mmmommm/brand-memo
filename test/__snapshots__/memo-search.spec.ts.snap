// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-search.vue <MemoSearch /> 1`] = `
<v-container-stub
  tag="div"
>
  <v-layout-stub
    tag="div"
  >
    <searchbar-stub />
     
    <searchbutton-stub />
  </v-layout-stub>
   
  <v-layout-stub
    tag="div"
    wrap=""
  >
    <memocard-stub
      memo="[object Object]"
    />
  </v-layout-stub>
</v-container-stub>
`;
